docker/Dockerfile

```diff
@@ -0,0 +1,41 @@
+# Use the Python Alpine base image
+FROM python:3.12-alpine
+
+# Set the working directory
+WORKDIR /app
+
+# Set environment variables
+ENV PYTHONUNBUFFERED=1 \
+    POETRY_VERSION=1.8.3 \
+    POETRY_HOME="/opt/poetry" \
+    PATH=$PATH:/opt/poetry/bin
```

Comment:
@vasylboliuk, it is more often used to divide env variable definitions for several layers, so if one env is changed, others don't require rehashing. Please divide these variables into several ENV and for PATH variable reuse the POETRY_HOME variable

Reply:
Done